app/views/layouts/_header.haml

```diff
@@ -1,11 +1,11 @@
 .header-bar
   %div.header-item
-    = image_tag('print-logo-white', class: 'header-logo')
```

Comment:
Is `print-logo-white` being used anywhere else in the code? If this was the only place, it should probably be deleted.

Reply:
hmmm, I don't seem to have that line of code even after pulling.
This is what I see: 
![Screenshot from 2019-06-12 13-42-25](https://user-images.githubusercontent.com/13788498/59317661-01c61100-8d18-11e9-88e0-9d531abbe1a9.png)

Reply:
So when you removed the image tag, you removed `print-logo-white` which is the logo that used to be in the header. I was just suggesting that you do a global search in the repo for `print-logo-white` and if it isn't used anywhere else, delete the image file from `app/assets/images`. If it is used somewhere else in the code, ignore this. I suspect that was the only place though.

Reply:
Yup, you're right! Not used in anywhere else, so have deleted the image.